ehealthid-rp/src/test/java/com/oviva/ehealthid/relyingparty/ws/AuthEndpointTest.java

```diff
@@ -29,13 +31,15 @@ class AuthEndpointTest {
 
   private static final URI BASE_URI = URI.create("https://idp.example.com");
   private static final URI REDIRECT_URI = URI.create("https://myapp.example.com");
+  private static final PrometheusMeterRegistry prometheusMeterRegistry =
```

Comment:
If you like, we can stub that out, however I feel that since this is a lightweight implementation with no externalities this could be fine.

Also, please let me know if you want a test case to validate the counter incrementing behavior.

Reply:
That should be fine. Alterntatively you can `spy(new PrometheusMeterRegistry(PrometheusConfig.DEFAULT))` on it with Mockito to verify that it was called.
Or simpler just instantiate this with `mock(PrometheusMeterRegistry.class)`.

Reply:
In this case, I would leave it as is for now. Should we want to verify something about the counter later, we can mock the registry and the provisioned meters at a later point.